libraries/AP_Camera/AP_Camera.h

```diff
@@ -68,6 +69,9 @@ class AP_Camera {
 
     static const struct AP_Param::GroupInfo        var_info[];
 
+    // get update about mode
```

Comment:
Sorry, but this comment isn't good: this isn't a getter, but a setter, and not for any mode, but just for Auto. Something like *set if vehicle is in AUTO mode* is better.